sources/Metal/Renderers/iTermCursorGuideRenderer.h

```diff
@@ -1,17 +1,18 @@
 #import <Foundation/Foundation.h>
 
 #import "iTermMetalCellRenderer.h"
+#import "VT100GridTypes.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
 @interface iTermCursorGuideRendererTransientState : iTermMetalCellRendererTransientState
-// set to -1 if cursor's row is not currently visible.
-- (void)setRow:(int)row;
+- (void)setCursorCoord:(VT100GridCoord)coord within:(VT100GridSize)bounds;
```

Comment:
It's not necessary to pass the screen size in cells to a subclass of `iTermMetalCellRendererTransientState` because it can access it from `self.cellConfiguration.gridSize`.

Reply:
Done.